nestkernel/nest_time.cpp

```diff
@@ -77,7 +78,7 @@ Time::compute_max()
   const tic_t tmax = std::numeric_limits< tic_t >::max();
 
   tic_t tics;
-  if ( lmax < tmax / Range::TICS_PER_STEP ) // step size is limiting factor
+  if ( lmax < tmax * Range::TICS_PER_STEP_INV ) // step size is limiting factor
```

Comment:
@apeyser @heplesser I'd vote for readability. In `struct range{}` there are e.g. definitions for `STEPS_PER_MS` and `MS_PER_STEP`. Wouldn't it be consequent to have `STEPS_PER_TIC` ?